operations/products.md

```diff
@@ -29,351 +32,361 @@ Returns all products offered together with the specified services. Note this ope
+| `ImageIds` | array of string | optional | Unique identifier of the product image. |
+| `Ordering` | integer | required | Order value for presentation purposes. |
+| `ExternalIdentifier` | string | optional, max length 255 characters | Identifier of the product from external system. |
+| `CreatedUtc` | string | required | Creation date and time of the product in UTC timezone in ISO 8601 format. |
+| `UpdatedUtc` | string | required | Last update date and time of the product in UTC timezone in ISO 8601 format. |
+| `ConsumptionMoment` | [Product consumption moment](products.md#product-consumption-moment) | required | Consumption moment of the product. |
+| ~~`IsDefault`~~ | ~~boolean~~ | ~~required~~ | **Deprecated!** |
+| ~~`Name`~~ | ~~string~~ | ~~optional~~ | ~~Name of the product. **Deprecated!** Please use Names~~ **Deprecated!** Use `Names` instead.|
+| ~~`ExternalName`~~ | ~~string~~ | ~~optional~~ | ~~Name of the product meant to be displayed to customer. **Deprecated!** Please use ExternalNames~~ **Deprecated!** Use `ExternalNames` instead.|
+| ~~`ShortName`~~ | ~~string~~ | ~~optional~~ | ~~Short name of the product. **Deprecated!** Please use ShortNames~~ **Deprecated!** Use `ShortNames` instead.|
+| ~~`Description`~~ | ~~string~~ | ~~optional~~ | ~~Description of the product. **Deprecated!** Please use Descriptions~~ **Deprecated!** Use `Descriptions` instead.|
+| ~~`Charging`~~ | ~~[Product charging mode](products.md#product-charging-mode)~~ | ~~required~~ | **Deprecated!** |
+| ~~`Posting`~~ | ~~[Product posting](products.md#product-posting)~~ | ~~required~~ | **Deprecated!** |
+| ~~`Pricing`~~ | ~~[Product pricing](products.md#product-pricing)~~ | ~~required~~ | **Deprecated!** Use `Price` instead.|
+
+#### Product charging mode
+
+* `Once`
+* `PerTimeUnit`
+* `PerPersonPerTimeUnit`
+* `PerPerson`
+
+#### Product posting
+
+* `Once`
+* `Daily`
+
+#### Product posting mode
+
+* `Once`
+* `PerTimeUnit`
+
+#### Product options
+Options of the product.
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `BillAsPackage` | boolean | required | Product should be displayed as part of a package. |
+
+#### Product classifications
+Classifications of the service.
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Food` | boolean | required | Product is classified as food. |
+| `Beverage` | boolean | required | Product is classified as beverage. |
+| `Wellness` | boolean | required | Product is classified as wellness. |
+| `CityTax` | boolean | required | Product is classified as city tax. |
+
+#### Product pricing
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Discriminator` | [Product pricing data discriminator](products.md#product-pricing-data-discriminator) | required | Determines type of value. |
+| `Value` | object | required | Structure of object depends on `Discriminator`. |
+
+#### Product pricing data discriminator
+
+* `Absolute`
```

Comment:
Absolute product type definition seems missing

Reply:
Same as comment below, it wasn't even in the docs.